admins/pageflow/entry.rb

```diff
@@ -169,23 +210,67 @@ def permitted_params
 
       private
 
+      def account_policy_scope
+        AccountPolicy::Scope.new(current_user, Pageflow::Account)
+      end
+
+      def theming_policy_scope
+        ThemingPolicy::Scope.new(current_user, Pageflow::Theming)
+      end
```

Comment:
Can these methods also be used in some of the cases above where we now reference the scope classes directly?